src/main/webapp/app/overview/course-card.scss

```diff
@@ -2,7 +2,8 @@
     height: 320px;
     cursor: pointer;
     .card-header {
-        height: 60px;
+        min-height: 60px;
```

Comment:
We can increase the `min-height` a bit if we want all cards to have the same header hight.